cognite/neat/rules/exporter/dm/rules2dms.py

```diff
@@ -1,2 +1,315 @@
-"""Exports rules to CDF Data Model Storage.
```

Comment:
Why is this in a submodule `dm` . Suggest moving it up one level as it is only one file.

Reply:
moved on step up, it was purely for org purpose